### ISSUE TITLE
Missing docs for enterprise connections

### ISSUE THREAD

**Issue by glena** [auth0/docs#682]
Several links to enterprise connections on IdP doc (https://auth0.com/docs/identityproviders) are broken and points nowhere:
- https://auth0.com/docs/connections/enterprise/ip-address
- https://auth0.com/docs/connections/enterprise/ldap (does it should point to AD one? https://auth0.com/docs/connections/enterprise/active-directory)
- https://auth0.com/docs/connections/enterprise/ping-federate
- https://auth0.com/docs/connections/enterprise/samlp (does it should point to the SAML doc? https://auth0.com/docs/saml-configuration)
- https://auth0.com/docs/connections/enterprise/ws-fed

**Comment by glena** [comment]
https://auth0.com/docs/connections/enterprise/google-apps

**Comment by Annyv2** [comment]
Relates to https://github.com/auth0/docs/issues/94 https://github.com/auth0/docs/issues/92 https://github.com/auth0/docs/issues/93 https://github.com/auth0/docs/issues/95 https://github.com/auth0/docs/issues/96 https://github.com/auth0/docs/issues/97

**Comment by Catografix** [comment]
@ntotten All these docs exists in the repo, dont know why they are not displaying live. They used to work. The ping-federate is blank - Im waiting to hear from them for access.

**Comment by ntotten** [comment]
@Catografix Yeah, they are deliberately blank - and hidden. We are using them for metadata only. We should add the full information though and make them public. right now they are private as noted by the config at the top of the file `public: false`

**Comment by mpaktiti** [comment]
Missing docs:

- [ ] https://auth0.com/docs/connections/enterprise/ip-address
- [ ] https://auth0.com/docs/connections/enterprise/ldap (same with AD one? https://auth0.com/docs/connections/enterprise/active-directory)
- [ ] https://auth0.com/docs/connections/enterprise/ping-federate
- [ ] https://auth0.com/docs/connections/enterprise/ws-fed

**Comment by dmark** [comment]
Anything I can do to help with this? The above docs lgtm, though LDAP and AD could maybe use a refactor given the amount of overlap between them.
